test/libsolidity/ASTJSONTest.cpp

```diff
@@ -237,8 +239,6 @@ void ASTJSONTest::printSource(ostream& _stream, string const& _linePrefix, bool
 {
 	for (auto const& source: m_sources)
 	{
-		if (m_sources.size() > 1 || source.first != "a")
-			_stream << _linePrefix << "// ---- SOURCE: " << source.first << endl << endl;
```

Comment:
With this removed, `isoltest` prints the source without delimiters, so it looks like a single file:

``` solidity
    /**This contract is empty*/ contract C {}

    /**This contract is empty
     and has a line-breaking comment.*/
    contract C {}

    contract C {
      /** Some comment on state var.*/ uint public state;
      /** Some comment on Evt.*/ event Evt();
      /** Some comment on mod.*/ modifier mod() { _; }
      /** XXX*/ function fn() public {}
    }
```
Note: I replaced the last comment with `XXX` to have the test fail.

Reply:
Good point, I'll add it back and adjust it